x/slashing/client/rest/tx.go

```diff
@@ -25,6 +26,24 @@ type UnjailReq struct {
 	BaseReq rest.BaseReq `json:"base_req" yaml:"base_req"`
 }
 
+type stdTx struct { // nolint: deadcode unused
```

Comment:
is this wrapper required?

Reply:
is there way to import stdtx without using it? since its in a comment go removes it from the import list